.github/workflows/tests.yml

```diff
@@ -0,0 +1,37 @@
+name: Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 13.x, 14.x]
+    
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: Install deps
+      run: yarn
```

Comment:
You could also cache and restore the dependencies by using [actions/cache](https://github.com/actions/cache/blob/main/examples.md#node---yarn-2), but I think it is only worth doing if the dep install is lengthy and slow.